rgbd_image_buffer/include/rgbd/image_buffer/image_buffer.h

```diff
@@ -59,21 +59,21 @@ class ImageBuffer
      * @param[out] image rgbd image to write the next image to. Iff a next image is found
      * @param[out] sensor_pose will be filled with the sensor pose corresponding to the next image. Iff a next image is found
      * @param[in] timeout_sec maximum duration to block.
-     * @param[in,opt] check_rate Check for a new image on this frequency. In case =0, set to 10 Hz
+     * @param[in] check_rate Check for a new image on this frequency. In case =0, set to 10 Hz
      * @return whether or not a next image was available within the timeout duration.
      */
-    bool waitForRecentImage(rgbd::ImageConstPtr& image, geo::Pose3D& sensor_pose, double timeout_sec, double check_rate = 10.);
```

Comment:
why remove this default?

Reply:
You can't have two functions with the same arguments. A function with default arguments, is the same as the function without the default arguments. So that would result in two `waitForRecentImage` with an `ImagePtr`, `Pose` and `double`.